.github/workflows/c.yml

```diff
@@ -3,7 +3,23 @@ name: C bindings
 on:
    push:
       branches: [ RC_1_2 RC_2_0 master ]
+      paths:
+         - src/**
+         - include/**
+         - deps/**
+         - Jamfile
+         - Jamroot.jam
+         - bindings/c/**
+         - "!bindings/c/tools/**"
    pull_request:
+      paths:
+         - src/**
+         - include/**
+         - deps/**
+         - Jamfile
+         - Jamroot.jam
+         - bindings/c/**
+         - "!bindings/c/tools/**"
```

Comment:
I take it these are python tools. I think this might be a bit too granular. I'm worried about there being a mistake in these cut-outs, so the simpler the better I think

Reply:
Hm, these are all files to build the c bindings (I believe).

I think you may have read this as a disallow-list ("don't run workflow if only these files modified"), when it's actually an allow-list ("only run workflow if any of these files modified").

The disallow-list is probably the safer option. I'll change it to this.

Reply:
oh, yeah. I assumed the `!`-prefix meant "not"